plugin/trino-resource-group-managers/src/test/java/io/trino/plugin/resourcegroups/db/TestDbResourceGroupsFlywayMigration.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.resourcegroups.db;
+
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+import static org.testng.Assert.assertEquals;
+
+@Test(singleThreaded = true)
+public class TestDbResourceGroupsFlywayMigration
+{
+    private TestingMysqlServer mysqlServer;
+
+    @BeforeClass
+    public final void setup()
+    {
+        mysqlServer = new TestingMysqlServer()
+                .withDatabaseName("resource_groups")
+                .withUsername("test")
+                .withPassword("test");
+        mysqlServer.start();
+        createOldSchema();
```

Comment:
I think that for completeness, we should have tests covering all pre-existing conditions:

- empty database
- database with the old schema present
- database without the old schema, but with some other random table

Flyway behaves differently in each of these scenarios.

Reply:
Good idea. Updated to add tests for these conditions.